sycl/unittests/Extensions/CommandGraph.cpp

```diff
@@ -20,6 +21,62 @@
 using namespace sycl;
 using namespace sycl::ext::oneapi;
 
+namespace {
```

Comment:
I think a brief comment might be useful here.

Reply:
I added some comments to this namespace.